### PR TITLE
Project: Weather App

### PR DESCRIPTION
**Pull request by cvilla714**
The project uses an API from Accuweather
once the name of the city is inserted in the form the following information is displayed

- the name of the city
- time zone
- forecast 
- temperature
-the picture will change between night and day depending on the weather

### CODE REVIEW

File: src/forecast.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const key = 'Fa1tORczSKnflphuiPFCKU303Z04wWn9';
+
+const getWeather = async (locationid) => {
```
Comment (christianotieno) [inline src/forecast.js:3]: ## Add a new module for the `toggle` functionality
- [x] This project is required to have toggle functionality in place. There is an excellent effort in displaying the temperature in both versions (celsius and Fahrenheit). 
temp-toggle
In this instance, there were two options to go about this:

- Either make two different requests for the `Celsius data` and the `Fahrenheit data` OR make only one request and use math to convert it to the other unit as expressed here in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app). 
- It is great you chose to go with the former, now let's add the toggle functionality. As a user, I need to toggle in between the `temperature` values.

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @cvilla714 👋 

The job done here is great. 👏 
Below, 👇 I have pointed out an issue with the implementation of the `toggle` functionality.
### Happy coding 💻 
#### Christian Otieno (CR)

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Nice work so far. 👍 

## However, there are some changes required 👀 🧰 🔨 🔧 

- Good work adding a toggle button. However, the Live Demo link on your README file needs to be updated to reflect this change.

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by voscarmv — APPROVED** [review]
# Project status: Approved ✨ 😎👍 💯 ✔️
### You may merge this branch and move forward ⛙🛣

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**